pallets/astar-xcm-benchmarks/src/fungible/assets/benchmarking.rs

```diff
@@ -0,0 +1,249 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Astar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Astar is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+use super::*;
+use crate::{account_and_location, new_executor, AssetTransactorOf, XcmCallOf};
+use frame_benchmarking::{benchmarks_instance_pallet, BenchmarkError, BenchmarkResult};
+use frame_support::traits::fungible::Inspect;
+use sp_runtime::traits::Zero;
+use sp_std::{prelude::*, vec};
+use xcm::latest::{prelude::*, Weight};
+use xcm_executor::traits::{Convert, TransactAsset};
+
+benchmarks_instance_pallet! {
+    where_clause { where
+        <
+            <
+                T::TransactAsset
+                as
+                Inspect<T::AccountId>
+            >::Balance
+            as
+            TryInto<u128>
+        >::Error: sp_std::fmt::Debug,
+
+            <   T::TransactAsset
+                as
+                Inspect<T::AccountId>
+            >::Balance : Into<u128>
+
+    }
```

Comment:
Why use old `v1` benchmarks if new `v2` has been available for many months?

Reply:
changed to v2.